quickstart/known-limitations.md

```diff
@@ -4,10 +4,6 @@
 
 Once provisioned, neither Mayastor Disk Pools nor Mayastor Volumes can be re-sized. A Mayastor Pool can have only a single block device as a member. Mayastor Volumes are exclusively thick-provisioned.
 
-## Snapshots and Clones
-
-Mayastor has no snapshot or cloning capabilities.
```

Comment:
Currently we can only create a snapshot and clone for a single replica volume.

cc @avishnu @tiagolobocastro Should we mention here that, although we can create a single replica clone/restore, we can still scale it up later?